src/main/java/com/microsoft/jenkins/containeragents/aci/AciCloud.java

```diff
@@ -55,9 +63,11 @@
     public AciCloud(String name,
                     String credentialsId,
                     String resourceGroup,
-                    List<AciContainerTemplate> templates) {
+                    List<AciContainerTemplate> templates,
+                    String logAnalyticsCredentialsId) {
```

Comment:
Optional parameters must be set via setters (annotated with DataBoundSetter)
https://www.jenkins.io/doc/developer/plugin-development/pipeline-integration/#constructor-vs-setters